<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignMessageDialog</class>
 <widget class="QDialog" name="SignMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>413</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sign or verify message</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/images/cryptonote</normaloff>:/images/cryptonote</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ClosePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="m_signTab">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <attribute name="title">
       <string>Sign message</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="m_messageEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Signature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="m_signatureEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_verifyTab">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <attribute name="title">
       <string>Verify message</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="m_addressLabel">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_addressEdit">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_verifySignatureLabel">
           <property name="text">
            <string>Signature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="m_verifySignatureEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_verifyMessageLabel">
           <property name="text">
            <string>Message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="m_verifyMessageEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="m_verificationResult">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ClosePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SignMessageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_messageEdit</sender>
   <signal>textChanged()</signal>
   <receiver>SignMessageDialog</receiver>
   <slot>messageChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SignMessageDialog</receiver>
   <slot>verifyMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_verifySignatureEdit</sender>
   <signal>textChanged()</signal>
   <receiver>SignMessageDialog</receiver>
   <slot>verifyMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_verifyMessageEdit</sender>
   <signal>textChanged()</signal>
   <receiver>SignMessageDialog</receiver>
   <slot>verifyMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>SignMessageDialog</receiver>
   <slot>changeTitle(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
